library/src/main/java/com/novoda/downloadmanager/LiteDownloadMigrationService.java

```diff
@@ -52,6 +58,15 @@ public void onCreate() {
         }
 
         binder = new MigrationDownloadServiceBinder();
+        channelCreator = new MigrationNotificationChannelCreator(getResources());
```

Comment:
Neither the `MigrationNotificationChannelCreator ` or the `MigrationNotification` can be injected by the client, that means that the client has no choice but to show in the way encoded here the migrations notifications right? If we allow the client to specify its own, then this library just needs to provide a basic good-enough implementation.

Reply:
I will be adding a way for the client to inject this, sorry I should have mentioned. This will be done in a follow-up PR to keep things small and contained 😄